.github/workflows/docker-build-push-github.yaml

```diff
@@ -22,6 +22,10 @@ on:
         required: false
         type: string
         default: ${{ github.sha }}
+      platforms:
+        description: "define multi-platform images"
```

Comment:
example would be great, i am not sure if we would need a matrix functionality here or if it will be enough to just send more params to buildx